app/models/bet.rb

```diff
@@ -22,4 +22,8 @@ def winner_total
   def amount_owed(lost_bet, won_bet)
     lost_bet.amount_bet * (won_bet.amount_bet / winner_total).round(2)
   end
+
+  def bet_total
+    user_bets.map(&:amount_bet).sum
```

Comment:
This is correct.

You could also do this with `user_bets.pluck(:amount_bet).sum` Which doesnt instantiate UserBet objects and performs the sum at the database level, instead of the application level